### ISSUE TITLE
Crash while unfollowing users

### ISSUE THREAD

**Issue by Jannik098** [alexal1/Insomniac#350]
Kind of every third users the bot is trying to unfollow I get the crash as you can see in the screenshot. After this the bot restarts Instagram and most of the time can successfully unfollow some users and then gets the same crash again.
![image](https://user-images.githubusercontent.com/58641072/122782805-4eeaff00-d2b1-11eb-9aa4-3fd9a3dbdefe.png)

**Comment by Jannik098** [comment]
Here is the crash zip file 
[Crash-2021-06-21-16-48-38.zip](https://github.com/alexal1/Insomniac/files/6687583/Crash-2021-06-21-16-48-38.zip)

**Comment by alexal1** [comment]
Hey @Jannik098, did you mention what happens on the screen at this moment?
I see that there's "Back to the followings list" message in logs, but according to screenshot it didn't go there. Maybe some dialog was opened?

Note that you make the script record a video for you:
```
pip3 install -U "uiautomator2[image]" -i https://pypi.doubanio.com/simple

Run Insomniac with --debug:
python start.py ... --debug
```

**Comment by noovy** [comment]
Same issue happen to me. Script after unfollowing command didn't come back to list with them.

```
[06/21 15:04:01] Unfollowing @mashry.me...
[06/21 15:04:01] Sleep for 1.96 seconds
[06/21 15:04:11] Back to the followings list.
[06/21 15:04:12] Unfollowed @mashry.me
[06/21 15:04:12] Session progress: 1 unfollowed for the entire session
[06/21 15:04:12] Sleep for 1.12 seconds
[06/21 15:04:15] Next item not found: probably reached end of the screen.
[06/21 15:04:15] Need to scroll now
[06/21 15:04:26] Error - -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd
Traceback (most recent call last):
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\device_facade.py", line 518, in scroll
    self.viewV2.scroll.toEnd(max_swipes=1)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\_selector.py", line 574, in __call__
    return jsonrpc.scrollToEnd(selector, self.vertical,
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 452, in __call__
    return self.server._jsonrpc_retry_call(self.method, params,
  File "C:\Users\micno\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 459, in _jsonrpc_retry_call
    return self._jsonrpc_call(*args, **kwargs)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 531, in _jsonrpc_call
    raise err
uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\safely_runner.py", line 19, in wrapper
    func(*args, **kwargs)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\action_runners\unfollow\__init__.py", line 78, in job
    unfollow(device=device_wrapper.get(),
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\action_runners\unfollow\action_unfollow.py", line 91, in unfollow
    iterate_over_my_followings(device, iteration_callback, iteration_callback_pre_conditions)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\actions_impl.py", line 668, in iterate_over_my_followings
    _iterate_over_my_followers_or_followings(device,
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\actions_impl.py", line 718, in _iterate_over_my_followers_or_followings
    list_view.scroll(DeviceFacade.Direction.BOTTOM)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\device_facade.py", line 520, in scroll
    raise DeviceFacade.JsonRpcError(e)
insomniac.device_facade.DeviceFacade.JsonRpcError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd


Error - -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd
Traceback (most recent call last):
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\device_facade.py", line 518, in scroll
    self.viewV2.scroll.toEnd(max_swipes=1)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\_selector.py", line 574, in __call__
    return jsonrpc.scrollToEnd(selector, self.vertical,
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 452, in __call__
    return self.server._jsonrpc_retry_call(self.method, params,
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 459, in _jsonrpc_retry_call
    return self._jsonrpc_call(*args, **kwargs)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\uiautomator2\__init__.py", line 531, in _jsonrpc_call
    raise err
uiautomator2.exceptions.UiObjectNotFoundError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\safely_runner.py", line 19, in wrapper
    func(*args, **kwargs)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\action_runners\unfollow\__init__.py", line 78, in job
    unfollow(device=device_wrapper.get(),
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\action_runners\unfollow\action_unfollow.py", line 91, in unfollow
    iterate_over_my_followings(device, iteration_callback, iteration_callback_pre_conditions)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\actions_impl.py", line 668, in iterate_over_my_followings
    _iterate_over_my_followers_or_followings(device,
  File "C:\Users\mxxAppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\actions_impl.py", line 718, in _iterate_over_my_followers_or_followings
    list_view.scroll(DeviceFacade.Direction.BOTTOM)
  File "C:\Users\mxx\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\insomniac\device_facade.py", line 520, in scroll
    raise DeviceFacade.JsonRpcError(e)
insomniac.device_facade.DeviceFacade.JsonRpcError: -32001 Jsonrpc error: <androidx.test.uiautomator.UiObjectNotFoundException> data: UiSelector[CLASS=android.widget.ListView, RESOURCE_ID=android:id/list], method: scrollToEnd

```

**Comment by alexal1** [comment]
@noovy so what exactly happened?
> [06/21 15:04:11] Back to the followings list.

Back press just didn't work or it closed something another?

**Comment by noovy** [comment]
Just back press, there was no popup from instagram.

**Comment by alexal1** [comment]
@noovy can you record a video please? Right now I don't understand how to fix this.

**Comment by noovy** [comment]
@alexal1 Here is clip of what happen.

https://gofile.io/d/84nKkw

**Comment by Jannik098** [comment]
I was not able to make the debug screen record work. But for me it's same as @noovy video, it crashes and goes to my phones homescreen and then tries the same process again (going into Instagram, following tab, ...).

**Comment by alexal1** [comment]
Fixed in `v3.7.18`
